contrib/fwupd.spec.in

```diff
@@ -120,10 +120,6 @@ Requires: shared-mime-info
 Requires: dbxtool
 %endif
 
-%if 0%{?rhel} > 7 || 0%{?fedora} > 28
-Recommends: python3
```

Comment:
Oh yeah so can you make this Recommends fwupd-utils? So on Fedora and rhel it installs them still?

Reply:
How about making it weaker and using `Suggests`? I think the great majority of people won't need the extra tools.

Reply:
I think you're right, but we're talking what 3kb?
Maybe install them by default without a subpackage and make python3 a suggests?

Reply:
>  Maybe install them by default without a subpackage and make python3 a suggests?

Ya, I though this what was happening now, but apparently rpm sees the python files and then slaps on a NEEDS PYTHON dep automatically. Maybe the real fix is stopping rpm from doing that.

Reply:
Alternative proposal in https://github.com/fwupd/fwupd/pull/2226

Reply:
That approach is much better thanks.